kernel/data-pipeline/core/src/test/java/org/apache/shardingsphere/data/pipeline/core/check/consistency/algorithm/fixture/FixturePipelineSQLBuilder.java

```diff
@@ -76,6 +76,11 @@ public String buildChunkedQuerySQL(final String schemaName, final String tableNa
         return "";
     }
     
+    @Override
+    public String buildChunkedQueryUniqueKeySQL(final String schemaName, final String tableName, final String uniqueKey, final boolean firstQuery) {
+        return String.format("SELECT %s FROM %s ORDER BY %s ASC LIMIT ?", uniqueKey, tableName, uniqueKey);
+    }
```

Comment:
If it's not necessary to return real SQL, then we could remove it, or else we need to update it

Reply:
Changed to "", just use to Just to adapt unit tests